app/views/devise/registrations/new.html.erb

```diff
@@ -38,46 +38,48 @@
     <div class="card-body">  
       <h5 class="card-title">User</h5>
         <div>
-             <%= f.label :picture %>
-             <%= f.text_field :picture, autofocus: true %>
+           <%= f.label :picture %>
+           <%= f.text_field :picture, autofocus: true %>
```

Comment:
This form looks the same as the form used in `app/views/devise/registrations/edit.html.erb`. To avoid having to re-do all of the corrections noted above and to make your code more DRY ("don't repeat yourself"), please consider moving all of the form content into a separate partial view (`_form.html.erb`) and then rendering that form in both the `new` and `edit` views.